src/middlewared/middlewared/plugins/smb.py

```diff
@@ -475,22 +476,22 @@ def getparm(self, parm, section):
 
     @accepts(Dict(
         'smb_update',
-        Str('netbiosname'),
-        Str('netbiosname_b'),
+        Str('netbiosname', validators=[Range(max=120)]),
+        Str('netbiosname_b', validators=[Range(max=120)]),
         List('netbiosalias', default=[]),
-        Str('workgroup'),
-        Str('description'),
+        Str('workgroup', validators=[Range(max=120)]),
+        Str('description', validators=[Range(max=120)]),
         Bool('enable_smb1'),
-        Str('unixcharset'),
+        Str('unixcharset', validators=[Range(max=120)]),
```

Comment:
Am I the only one, who is annoyed with this noise?
`netbiosname` is up to 120 characters, but `netbiosalias` can be of any length, is this what this code says?